lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js

```diff
@@ -58,23 +57,16 @@ $(document).on('click', '.js-kudos-button', function(event) {
   const dataKid = $el.attr('data-kid');
   const method = dataKid ? 'DELETE' : 'POST';
 
-  // if (setting('dsKudosReactions.sendToRogue')) {
-  //   makeKudosRequestToRogue(setting('dsKudosReactions.northstarId'), setting('dsKudosReactions.postId'))
-  // } else {
   makeKudosRequest($el, dataKid)
     .done((response) => {
       const kid = get(response, '0.kid', '');
-
-      if (method === 'POST' && !kid) {
-        return;
-      }
-
+      // if (method === 'POST' && !kid) {
+      //   return;
+      // }
```

Comment:
The problem with this conditional is that if the user has liked this already and tries to unlike it, since it doesn't know about the `kid` in [this line](https://github.com/DoSomething/phoenix/blob/dev/lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js#L58), it thinks it's a `POST` request instead of a `DELETE`.